app/tests/api/test_login.py

```diff
@@ -0,0 +1,102 @@
+from typing import Dict
+from sqlalchemy.orm import Session
+
+from fastapi.testclient import TestClient
+
+from app import crud
+from app.schemas.user import UserCreate
+from app.security import get_password_hash
+
+
+def test_get_access_token(client: TestClient, db: Session) -> None:
+    password = "PassWordCool"
+    
+    user_in = UserCreate(
+        email="user@user.com", password=password, username="other_username"
+    )
+    user = crud.user.create(db, obj_in=user_in)
+
+    login_data = {
+        "username": user.username,
+        "password": password
+    }
+    r = client.post("/login/token",  data=login_data)
+    tokens = r.json()
```

Comment:
You can create a function in this file doing those lines of code (taking a username and password as parameters and returning the tokens), to avoid duplication.